roles/storage/tasks/common_build_rings.yml

```diff
@@ -9,24 +9,40 @@
 #
 # This is necessary in case the cluster is halted.
 #
+
 - name: check if account.ring.gz already exists
   #stat: path=/etc/swift/{{ item }}.ring.gz
   command: ls /etc/swift/account.ring.gz
   register: rings_exist
   ignore_errors: true
   changed_when: rings_exist.rc > 0
 
+- name: do facts module to get latest information
+  setup:
+
 - debug: msg="{{ rings_exist.rc }}"
 
+- name: print the string with interpreted variables
+  ansible.builtin.debug:
+    msg: swift-ring-builder account.builder add r{{ region }}z{{ zone }}-{{ ansible_enp0s10.ipv4.address }}:6002R{{ ansible_enp0s16.ipv4.address }}:6002/{{ disk_prefix }}{{ item }} 100 
+  delegate_to: "{{ swift_proxy_server }}"
+  vars:
+    - ansible_ssh_user: vagrant
+    - ansible_ssh_private_key_file: /opt/swiftacular/.vagrant/machines/swift-proxy-01/virtualbox/private_key   
+  with_sequence: count={{ disks }}
+
+
 - name: build account ring
   command: swift-ring-builder account.builder add r{{ region }}z{{ zone }}-{{ ansible_enp0s10.ipv4.address }}:6002R{{ ansible_enp0s16.ipv4.address }}:6002/{{ disk_prefix }}{{ item }} 100
            chdir=/etc/swift
   delegate_to: "{{ swift_proxy_server }}"
+  vars:
```

Comment:
It seems this is the megic, if I put it once it "remmbers" it later in the file after 1 ssh.